Makefile

```diff
@@ -59,30 +81,28 @@ resolve:
 ensure:
 	@echo "Go modules present in component - omitting."
 
-dep-status:
+dep-status:	
 	@echo "Go modules present in component - omitting."
 
 mod-verify:
 	GO111MODULE=on go mod verify
 
- ## Run tests.
 test: fmt vet tidy
 	go test ./... -coverprofile cover.out
 
 build: fmt vet tidy ## Build manager binary.
 	go build -o bin/manager main.go
 
-lint: ## Run various linters
-	go version
-	golangci-lint version
-	GO111MODULE=on golangci-lint run
+lint: $(GOLANGCI_LINT) ## Check lint issues using `golangci-lint`
+	$(TOOLS_BIN_DIR)/golangci-lint run
+
+lint-fix: $(GOLANGCI_LINT) $(WSL)
```

Comment:
this target is called lint-auto fix in telemetry manager... please either change it here or there

Reply:
will be changed in telemetry-manager